kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelService.kt

```diff
@@ -103,9 +104,11 @@ class TunnelService : VpnService() {
                 addressIPv4: String,
                 addressIPv6: String,
                 dnsAddresses: String,
+                searchDomains: String,
             ) {
                 // init tunnel config
                 tunnelDnsAddresses = moshi.adapter<MutableList<String>>().fromJson(dnsAddresses)!!
+                tunnelSearchDomains = moshi.adapter<MutableList<String>>().fromJson(searchDomains)!!
```

Comment:
I think this will enforce the field to be required right? If true, we need to remember to ship the portal side before this PR merges / released. If we do that, will older connlibs still parse the new interface message just fine?

Reply:
I don't understand. The FFI serialisation has nothing to do with what the portal sends. Starting with this PR, we pass a list across the FFI layer here which is empty to begin with until the portal sends us search domains.

This PR can merge and even release completely on its own! :)

Reply:
Ah, understand now.